### PR TITLE
Add django migrations.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@Osmose r?

**Comment by Osmose** [comment]
r+, nice work!

File: django_jobvite/migrations/__init__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""
```
Comment (Osmose) [inline django_jobvite/migrations/__init__.py]: Is this auto-generated by Django? I've never seen this kind of file before.

Comment (glogiotatidis) [inline django_jobvite/migrations/__init__.py]: No I added this to make sure that it works with older south versions. South versions >= 1 look for `south_migrations` anywya.

**Pull request merged**